__init__.py

```diff
@@ -0,0 +1,28 @@
+import pGan_fSplice
+from maskgen.plugins import findPlugin
+import os
+
+"""JT code only used for plugin conversion"""
+def transform(img, source, target, **kwargs):
+	donor = kwargs['donor']
+	pGan_fSplice.loadModel(os.path.join(findPlugin('GAN_face_splice'),pGan_fSplice.PREDICTOR_PATH))
+	pGan_fSplice.splice_donor_recipient(source,donor,target)
+	return None, None
+
+def operation():
+	return {
+		'name': 'PasteSplice',
+		'category': 'Paste',
+		'software': 'GAN_face_splice',
+		'version': '0.1',
+		'arguments':{
+			'donor':{
```

Comment:
Add arguments:
 "donor rotated": {
          "type": "yesno",
          "description": "Enter yes if the donor is rotated during the paste operation"
        },
        "donor resized": {
          "type": "yesno",
          "description": "Enter yes if the donor is resized during the paste operation"
        },
        "donor cropped": {
          "type": "yesno",
          "description": "Enter yes if the donor is cropped the paste operation. Ideally, crop should occur as a SelectRegion operation just prior to donation."
        },
        "purpose": {
          "type": "list",
          "values": [
            "remove",
            "add",
            "blend"
          ],
          "description": "Purpose: remove an object, add an object."
        },
        "subject": {
          "type": "list",
          "values": [
            "people",
            "face",
            "natural object",
            "man-made object",
            "large man-made object",
            "landscape",
            "other"
          ],
          "description": "Subject catgeory pasted to image"
        }
set defaults:
subject:face
purpose:add
cropped:no
the rest = yes.